### ISSUE TITLE
I created an organization to fix problems known to raknet.

### ISSUE THREAD

**Issue by number201724** [OculusVR/RakNet#137]
repo: https://github.com/open-raknet/RakNet

**Comment by OvermindDL1** [comment]
I'm curious how this compares to https://github.com/SLikeSoft/SLikeNet, which seems to have the same goal?

**Comment by number201724** [comment]
Yes, but I see that his repo has no detailed commit log, only synced with SVN

**Comment by OvermindDL1** [comment]
Might be worth submitting a bug there then, I do agree that is quite irritating.  ^.^;

**Comment by Luke1410** [comment]
> 
> 
> Yes, but I see that his repo has no detailed commit log, only synced with SVN

That's of course right. We are going to switch this at some point to an automated mirror.
But since you are not the only one pointing this out, we'll change the procedure already now and will use a accumulated log from the corresponding SVN commits in future Git commits (see the recent commit where we just did that). That way it should be easier to follow what changed when you work with the Git repository.

Alternatively, you can always use the main repository https://www.slikesoft.com/svn/slikenet/trunk. If you don't wanna use SVN for that, you can also use Git with that one. Git works just fine even with SVN repositories (at least for the basic operations like checking out, updating, showing logs, etc.).

Another way to retrieve a list of what changed in a commit is to take a look at the corresponding changelog.txt file. Commits are usually accompanied by changes in that file, so you get an idea what a commit changed.

I hope this information helps a bit, but of course I can understand if you'd rather start from the original RakNet repository instead of using a fork to base your own work from.
You are always welcome to contribute patches/pull requests directly to SLikeNet of course, if you feel like that'd be a better investment of your time, but I certainly don't wanna discourage you from starting your own separate project. :-) Whatever way you decide to go: Wish you the best luck and don't hesitate to pick any changes you like from SLikeNet and feel free to use the free NatPunch service as well, of course.
